src/digest/digest.cr

```diff
@@ -133,13 +133,68 @@ abstract class Digest
     final dst
   end
 
+  # Writes the final digest output to `dst`.
```

Comment:
"writing to s.th." is typically associated with a stream-based operation, so I'm wondering if we can find better wording here.

Maybe this:
```suggestion
  # Puts the final digest output into `dst`.
```